submissions/madmaxWMFU/google-popup/index.html

```diff
@@ -0,0 +1,186 @@
+                                    <span class="textApp">YouTube</span>
+                                </a>
+                            </div>
+                            <div class="app">
+                                <a href="#">
+                                    <span class="iconApp" style="background-position:0 -1035px"></span>
+                                    <span class="textApp">News</span>
+                                </a>
+                            </div>
+                            <div class="app">
+                                <a href="#">
+                                    <span class="iconApp" style="background-position:0 -1104px"></span>
+                                    <span class="textApp">Gmail</span>
+                                </a>
+                            </div>
+                            <div class="app">
+                                <a href="#">
+                                    <span class="iconApp" style="background-position:0 -2139px"></span>
+                                    <span class="textApp">Contacts</span>
+                                </a>
+                            </div>
+                            <div class="app">
+                                <a href="#">
+                                    <span class="iconApp" style="background-position:0 -2001px"></span>
+                                    <span class="textApp">Drive</span>
+                                </a>
+                            </div>
+                            <div class="app">
+                                <a href="#">
+                                    <span class="iconApp" style="background-position:0 -828px"></span>
+                                    <span class="textApp">Calendar</span>
+                                </a>
+                            </div>
+                            <div class="app">
+                                <a href="#">
+                                    <span class="iconApp" style="background-position:0 -414px"></span>
+                                    <span class="textApp">G00gle+</span>
+                                </a>
+                            </div>
+                            <div class="app">
+                                <a href="#">
+                                    <span class="iconApp" style="background-position:0 -1656px"></span>
+                                    <span class="textApp">Translate</span>
+                                </a>
+                            </div>
+                            <div class="app">
+                                <a href="#">
+                                    <span class="iconApp" style="background-position:0 -1518px"></span>
+                                    <span class="textApp">Photos</span>
+                                </a>
+                            </div>
+                            <br>
```

Comment:
Using `<br>` for creating indent is a bad practice. Use the CSS property `margin` for this.

Reply:
Done!